extensions/amp-bind/0.1/test/test-bind-integration.js

```diff
@@ -101,4 +105,83 @@ describe.configure().retryOnSaucelabs().run('integration amp-bind', function() {
     });
   });
 
+  describe('amp-img integration', () => {
+    it('should change src when the src attribute binding changes', () => {
+      const changeImgSrcButton =
+        fixture.doc.getElementById('changeImgSrcButton');
+      const img = fixture.doc.getElementById('image');
+      expect(img.getAttribute('src')).to.equal('https://lh3.googleusercontent' +
+        '.com/5rcQ32ml8E5ONp9f9-Rf78IofLb9QjS5_0mqsY1zEFc=w300-h200-no');
+      changeImgSrcButton.click();
+      return waitForBindApplication().then(() => {
+        expect(img.getAttribute('src')).to.equal('https://lh3' +
+          '.googleusercontent.com/pSECrJ82R7-AqeBCOEPGPM9iG9O' +
+          'EIQ_QXcbubWIOdkY=w400-h300-no');
+      });
+    });
+
+    it('should NOT change src when new value is a blocked URL', () => {
+      const changeImgSrcButton =
+        fixture.doc.getElementById('changeImgSrcButton');
+      const img = fixture.doc.getElementById('image');
+      const originalSrc = 'https://lh3.googleusercontent.com/' +
+        '5rcQ32ml8E5ONp9f9-Rf78IofLb9QjS5_0mqsY1zEFc=w300-h200-no';
+      expect(img.getAttribute('src')).to.equal(originalSrc);
+      const newSrc = '__amp_source_origin';
```

Comment:
Prefer inlined literals over local vars in unit tests. IMO duplicating simple literals is more readable.

https://engdoc.corp.google.com/eng/doc/devguide/coding/testing/unit-testing-best-practices.shtml?cl=head#constants